src/main/java/core/basesyntax/supplier/ColorSupplier.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.supplier;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private enum FigureColorEnum {
+        RED, BLACK, WHITE, GREEN, YELLOW, BLUE, PURPLE;
+    }
```

Comment:
```suggestion
    private enum Color {
        RED, 
        BLACK, 
        WHITE, 
        GREEN, 
        YELLOW, 
        BLUE, 
        PURPLE;
    }
```
also better to move it to a separate class